scripts/configure-cluster/common/setup-postgres-operator.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -x
+
+
+install_postgres_operator(){
```

Comment:
Why aren't we adding this to `setup-operators.sh` script and creating a separate script for it instead?

Reply:
in a discussion with mzee we just thought have a different file would make sense because `setup-operators.sh` actually runs the functions as well which might have side-effects when someone is importing that file and just wants to install `postgres_operator`